Docs for user-land Vite (#3681)
wasp-lang/wasp: web/docs/migration-guides/migrate-from-0-21-to-0-22.md

---
title: Migration from 0.21.X to 0.22.X
---

import InstallInstructions from './_install-instructions.md'

<InstallInstructions version="0.22.0" />

## What's new in 0.22.X?

> Wasp 0.22 comes with a major overhaul of how the client is built. User's project dir is now considered the client app directory. Vite is started in that directory and no longer in the generated `.wasp/out/web-app` directory.

Reviewer: ```suggestion
Wasp 0.22 comes with a major overhaul of how the client app is built. User's project directory is now considered the client app directory. This means that Vite is started in user's project directory instead of being generated in the `.wasp/out/web-app` directory.
```
